client/src/components/overview/GalleryView.jsx

```diff
@@ -31,30 +31,82 @@ const StyledOverviewGrid = styled.div`
     'OvDesc OvMeta'
 `;
 
+// Here is the axios request to receive the style information for a product
+const retrieveStyles = axios({
+  method: 'get',
+  url: 'https://app-hrsei-api.herokuapp.com/api/fec2/hr-rfe/products/37314/styles',
+  headers: {
+    Authorization: API_KEY
+  }
+  // data.data.results
+});
+
+// Here is the axios request to receive the product information
+const retrieveProductInfo = axios({
+  method: 'get',
+  url: 'https://app-hrsei-api.herokuapp.com/api/fec2/hr-rfe/products/37314/',
+  headers: {
+    Authorization: API_KEY
+  }
+  // data.data
+});
+
+// Here is the axios request to receive receive the ratings of a product
+const retrieveRatingInfo = axios({
+  method: 'get',
+  url: 'https://app-hrsei-api.herokuapp.com/api/fec2/hr-rfe/reviews/meta?product_id=37314',
+  headers: {
+    Authorization: API_KEY
+  }
+  // data.data.ratings
+})
+
+// This is the function to create the average rating
+function findAverageRating(ratings)  {
```

Comment:
We should write a test for this

Reply:
That is a ticket in trello! Set to complete 8/27